Assets/MixedRealityToolkit.Examples/Demos/Input/Scenes/TurnPointerOnOffExample/TurnPointersOnOff.cs

```diff
@@ -0,0 +1,134 @@
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+
+namespace Microsoft.MixedReality.Toolkit.Examples.Demos
+{
+    /// <summary>
+    /// This example demonstrates how to turn pointers on and off by 
+    /// specifying custom behaviors.
+    /// </summary>
+    public class TurnPointersOnOff : MonoBehaviour
+    {
+        List<Tuple<Type, Interactable>> pointerToggles = new List<Tuple<Type, Interactable>>();
+
+        public void SetRayEnabled(bool isEnabled)
+        {
+            PointerUtils.SetRayPointerBehavior(isEnabled ? PointerBehavior.Default : PointerBehavior.Off,
+                Handedness.Any);
+        }
+
+        public void SetGazeEnabled(bool isEnabled)
+        {
+            PointerUtils.SetGGVBehavior(isEnabled ? PointerBehavior.Default : PointerBehavior.Off);
+        }
+
+        public void SetGrabEnabled(bool isEnabled)
+        {
+            PointerUtils.SetGrabPointerBehavior(isEnabled ? PointerBehavior.Default : PointerBehavior.Off, Handedness.Any);
+        }
+
+        public void SetPokeEnabled(bool isEnabled)
+        {
+            PointerUtils.SetPokePointerBehavior(isEnabled ? PointerBehavior.Default : PointerBehavior.Off, Handedness.Any);
+        }
+
+        public void SetVR()
+        {
+            PointerUtils.SetPokePointerBehavior(PointerBehavior.Off, Handedness.Any);
+            PointerUtils.SetGrabPointerBehavior(PointerBehavior.Off, Handedness.Any);
+            PointerUtils.SetRayPointerBehavior(PointerBehavior.Default, Handedness.Any);
+            PointerUtils.SetGGVBehavior(PointerBehavior.Off);
+        }
+
+        public void SetFingerOnly()
+        {
+            PointerUtils.SetPokePointerBehavior(PointerBehavior.Default, Handedness.Any);
+            PointerUtils.SetGrabPointerBehavior(PointerBehavior.Off, Handedness.Any);
+            PointerUtils.SetRayPointerBehavior(PointerBehavior.Off, Handedness.Any);
+            PointerUtils.SetGGVBehavior(PointerBehavior.Off);
+        }
+
+        public void SetHoloLens1()
+        {
+            PointerUtils.SetPokePointerBehavior(PointerBehavior.Off, Handedness.Any);
+            PointerUtils.SetGrabPointerBehavior(PointerBehavior.Off, Handedness.Any);
+            PointerUtils.SetRayPointerBehavior(PointerBehavior.Off, Handedness.Any);
+            PointerUtils.SetGGVBehavior(PointerBehavior.Default);
+        }
+
+        public void SetHoloLens2()
+        {
+            PointerUtils.SetPokePointerBehavior(PointerBehavior.Default, Handedness.Any);
+            PointerUtils.SetGrabPointerBehavior(PointerBehavior.Default, Handedness.Any);
+            PointerUtils.SetRayPointerBehavior(PointerBehavior.Default, Handedness.Any);
+            PointerUtils.SetGGVBehavior(PointerBehavior.Off);
+        }
+
+        void Start()
+        {
+            IMixedRealityCapabilityCheck capabilityChecker = CoreServices.InputSystem as IMixedRealityCapabilityCheck;
+            if (capabilityChecker != null)
```

Comment:
@davidkline-ms just to check, is it the case that the input system should have deterministically started up by any arbitrary monobehavior's Start() function?

I know there were some others that tried accessing things but had to wait until Update() in order to guarantee that it was actually there.